src/components/PostGrid.tsx

```diff
@@ -34,53 +38,89 @@ const PostGrid = ({ data, user }: PostGridProps): JSX.Element => {
+        <div className="flex gap-[6px] ">
+            {data?.contributions[0] &&
+              <Avatar
+                contributor={data.contributions[0]?.contributor}
+                lastPr={dayjs(data.contributions[0]?.last_merged_at).fromNow()}/>}
 
+            {data?.contributions[1] &&
+              <Avatar
+                contributor={data.contributions[1]?.contributor}
+                lastPr={dayjs(data.contributions[1]?.last_merged_at).fromNow()}/>}
+            {data?.contributions[2] &&
+              <Avatar
+                contributor={data.contributions[2]?.contributor}
+                lastPr={dayjs(data.contributions[2]?.last_merged_at).fromNow()}/>}
+            {data?.contributions[3] &&
+              <Avatar
+                contributor={data.contributions[3]?.contributor}
+                lastPr={dayjs(data.contributions[3]?.last_merged_at).fromNow()}/>}
+            {data?.contributions[4] &&
+              <Avatar
+                contributor={data.contributions[4]?.contributor}
+                lastPr={dayjs(data.contributions[4]?.last_merged_at).fromNow()}/>}
+          {
+            data.contributions.length > 0 &&
+            <a className='text-grey-600 text-[14px] ml-[10px] mr-[10px]' href={`https://github.com/${data.full_name}/contributors`}>more...</a>
+          }
         </div>
 
-        <div className="flex">
-          <div
-            role="button"
-            tabIndex={0}
-            aria-pressed="false"
-            onClick={() => user_id ? handleVoteUpdateByRepo(votes, repo_id) : signIn({ provider: 'github' })}
-            onKeyDown={(e) => {
-              if (e.key === 'Enter') {
-                return user_id ? handleVoteUpdateByRepo(votes, repo_id) : signIn({ provider: 'github' });
-              }
-            }}
-            className="flex justify-center items-center text-base space-x-1 text-grey hover:text-saucyRed cursor-pointer transition-all duration-200"
-          >
-            <FaArrowAltCircleUp/>
-            <p className="font-bold">{votes}</p>
+        <div className='flex gap-[10px] items-center'>
+          <div className='flex gap-[5px] items-center'>
+            <div className='w-[14px] h-auto'>
+              <img src="/IssuesIcon.svg" alt="" />
+            </div>
+            <p className='text-[12px] text-grey130 font-semibold'>{numToKformat(issues)}</p>
+          </div>
+
+          <div className='flex gap-[5px] text-grey130 items-center'>
+            <div className='w-[14px] h-auto'>
+              <img src="/StarIcon.svg" alt="" />
+            </div>
+            <p className='text-[12px] font-semibold'>{numToKformat(stars)}</p>
           </div>
         </div>
       </div>
-
-      <a
-        className="w-full bg-transparent h-32 overflow-hidden rounded-md mb-2 flex justify-center"
-        href={getRepoLink(data.full_name)}
-        title={`Visit ${data.full_name}`}
-        target="_blank"
-        rel="noopener"
-      >
-        <img
```

Comment:
Why you remove this ? It is a true feature ? As I see it we have an issue for this `a`  #182

Reply:
@NsdHSO  The image tag is removed because we are no longer rendering any image here. Instead, we are rendering an HTML element to show the relevant data.